node/cli/src/cli.rs

```diff
@@ -79,19 +84,23 @@ pub struct Cli {
 	/// Instant block sealing
 	/// Blocks are triggered to be formed each time a transaction hits the validated transaction pool
 	/// Empty blocks can also be formed using the `engine_createBlock` RPC
-	///
-	/// Can only be used with frequency-no-relay and frequency-rococo-local feature flags
-	#[cfg(any(feature = "frequency-no-relay", feature = "frequency-rococo-local"))]
-	#[clap(long = "instant-sealing")]
-	pub instant_sealing: bool,
```

Comment:
Is there a way with clap to leave a message that the option has changed?

Aka if someone after this runs ./frequency --instant-sealing it wouldn't run, but would report that it has changed to be --sealing=instant ?

Reply:
I will check to see if it has support for deprecated parameters.  If not, I could trap them and display a warning message.

Reply:
Fixed.